main/src/ca/uwaterloo/flix/language/ast/TokenKind.scala

```diff
@@ -95,6 +97,8 @@ object TokenKind {
 
   case object InfixFunction extends TokenKind
 
+  case object KeywordAbsent extends TokenKind
```

Comment:
I believe we have removed Absent, Present, and relational_choose and relational_choose_star.